docs/MO_DG/prepare_model/MO_Python_API.md

```diff
@@ -0,0 +1,85 @@
+## Model Optimizer Python API {#openvino_docs_MO_DG_Python_API}
+
+- Model Optimizer has Python API for model conversion, which is represented by convert_model() method in openvino.tools.mo namespace.
```

Comment:
Should we need to use `convert_model()`?

Reply:
Done.